promgen/templates/promgen/project_detail_exporters.html

```diff
@@ -10,7 +10,9 @@
       <th colspan="4">Actions</th>
     </tr>
   {% for exporter in project.exporter_set.all %}
-    <tr {% if not exporter.enabled %}class="promgen-disabled"{% endif %}>
+    <tr {% if not exporter.enabled %}class="promgen-disabled"{% endif %}
```

Comment:
<img width="572" alt="mouseover labels" src="https://user-images.githubusercontent.com/89725/150749734-6c1e88b7-35a6-440b-b2fd-a4a477fc4128.png">

Even though we use the `title=` attribute in a few places to show extra information, I think in this case, additional labels are quite important, and we need to figure out how to make them always visible. Part of the challenge though is we do not currently have a lot of space there, so still thinking if we need to have a toggle collapse menu for it or not.

Reply:
Will button `Labels` be OK? Button

Reply:
Button with dropdown table added.